packages/expo-camera/ios/CameraViewModule.swift

```diff
@@ -182,23 +184,25 @@ public final class CameraViewModule: Module, ScannerResultHandler {
         #else // not simulator
         view.takePicture(options: options, promise: promise)
```

Comment:
Could we make `takePicture` async as well and then get rid of using a promise?

Reply:
We can make it async but we still need the promise to resolve when the delegate is called.